pkg/auth/api/user/user_actions.go

```diff
@@ -9,9 +9,9 @@ import (
 	"github.com/rancher/norman/parse"
 	"github.com/rancher/norman/types"
 	"github.com/rancher/rancher/pkg/auth/providerrefresh"
-	"github.com/rancher/rancher/pkg/auth/settings"
 	client "github.com/rancher/rancher/pkg/client/generated/management/v3"
 	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/settings"
```

Comment:
Any reason this got moved in imports? I believe it was ordered properly before the change.

Reply:
Ah, one is from `pkg/auth/settings`, the other is from `pkg/settings`. Took me a good bit staring at that to figure it out.